misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -322,7 +448,19 @@ const WebActionsContainer = (props: IState & IDispatchProps) => {
       </div>
     )
   } else {
-    return <SkeletonWebActions />
+    return (
+      <div>
+        <SkeletonWebActions />
+        <br />
+        <SkeletonWebActions />
+        <br />
+        <SkeletonWebActions />
+        <br />
+        <SkeletonWebActions />
+        <br />
+        <SkeletonWebActions />
```

Comment:
Are these webaction stubs? Is this saying that in the case when there are no webactions for a service, we'll see 5 stubbed web actions? If thats the behaviour maybe some text saying "No web actions" would suffice?

Reply:
These are [Blueprint Skeletons](https://blueprintjs.com/docs/#core/components/skeleton) that are equivalent in purpose to a loading spinner. If the Web Actions endpoint hasn't completed yet, then this will show up to pre-write the UI. Once the data is available, it just fills in. If there are no bound Web Actions then the screen would be empty (though this won't be the case since all containers must provide readiness, liveness, and status endpoints).

Reply:
Ah got it